scripts/redshift/transformers/cell_expression.py

```diff
@@ -108,48 +113,49 @@ def _parse_10x_bundle(self, bundle_dir):
         genes = [g.split("\t")[0].split(".", 1)[0] for g in
                  open(os.path.join(bundle_dir, "genes.tsv")).readlines()]
         barcodes = [b.strip() for b in open(os.path.join(bundle_dir, "barcodes.tsv")).readlines()]
-        genes_detected = (matrix != 0).sum(0)
-        cell_suspension_id = json.load(open(
-            os.path.join(bundle_dir, "cell_suspension_0.json")))["provenance"]["document_id"]
-        sequence_file_path = list(pathlib.Path(bundle_dir).glob("sequence_file_*.json"))[0]
-        lane_index = json.load(open(sequence_file_path))["lane_index"]
 
         # columns are cells, rows are genes
         expression_lines = []
         cell_lines = set()
+        cell_gene_counts = {}
+        cell_to_barcode = {}
+
         for i, j, v in zip(matrix.row, matrix.col, matrix.data):
             barcode = barcodes[j]
             gene = genes[i]
-            gene_count = genes_detected.item((0, j))
 
             # Just make up a cell id
             h = hashlib.md5()
-            h.update(keys["project_key"].encode())
-            h.update(keys["donor_key"].encode())
-            h.update(cell_suspension_id.encode())
+            bundle_uuid = pathlib.Path(bundle_dir).parts[-1]
+            h.update(bundle_uuid.encode())
```

Comment:
Oh so `bundle_uuid` (i.e. analysis table rows) have a 1:many relationship with cell and expression rows, and not many:many? This would simplify our DSS subscriptions given cellkey is unique to a bundle